cli/dinghy/check_env.rb

```diff
@@ -29,16 +29,16 @@ def expected
     }
   end
   
-  def shell
+  def is_fish_shell?
     begin
-        return `echo $SHELL`.strip
+        return !`echo $FISH_VERSION`.strip.empty?
```

Comment:
Hm I didn't notice during the original PR this weird pattern of shelling out to `echo` rather than just checking `ENV['SHELL']`. That's not new with your PR though so I'll just fix it later.